olive/olive_config.json

```diff
@@ -1,108 +1,344 @@
 {
     "passes": {
         "AppendPrePostProcessingOps": {
-            "module_path": "olive.passes.onnx.append_pre_post_processing_ops.AppendPrePostProcessingOps"
+            "module_path": "olive.passes.onnx.append_pre_post_processing_ops.AppendPrePostProcessingOps",
+            "supported_providers": [ "*" ],
```

Comment:
Any possibility to make those `*` as default value? So no need to add `*` for all passes.

Reply:
We could make "empty list" be equal to "ALL". I couldn't come up with a reason (or an example) where we would want an empty list to mean "None" but decided to use "*" to be explicit.

I can change that if there is a strong opinion among devs. I just preferred to be explicit rather than overloading the meaning of empty list.

Reply:
Explicit is fine.